### ISSUE TITLE
使用cpu可以正常训练模型，gpu却不行

### ISSUE THREAD

**Issue by jiangbossB** [PaddlePaddle/PaddleOCR#10930]
(ppocr) F:\python_project\PaddleOCR>python tools/train.py -c configs\det\my_det_mv3_db.yml
[2023/09/18 11:46:43] ppocr INFO: Architecture : 
[2023/09/18 11:46:43] ppocr INFO:     Backbone : 
[2023/09/18 11:46:43] ppocr INFO:         model_name : large
[2023/09/18 11:46:43] ppocr INFO:         name : MobileNetV3
[2023/09/18 11:46:43] ppocr INFO:         scale : 0.5
[2023/09/18 11:46:43] ppocr INFO:     Head :
[2023/09/18 11:46:43] ppocr INFO:         k : 50
[2023/09/18 11:46:43] ppocr INFO:         name : DBHead
[2023/09/18 11:46:43] ppocr INFO:     Neck :
[2023/09/18 11:46:43] ppocr INFO:         name : DBFPN
[2023/09/18 11:46:43] ppocr INFO:         out_channels : 256
[2023/09/18 11:46:43] ppocr INFO:     Transform : None
[2023/09/18 11:46:43] ppocr INFO:     algorithm : DB
[2023/09/18 11:46:43] ppocr INFO:     model_type : det
[2023/09/18 11:46:43] ppocr INFO: Eval :
[2023/09/18 11:46:43] ppocr INFO:     dataset :
[2023/09/18 11:46:43] ppocr INFO:         data_dir : ./data/50000/test
[2023/09/18 11:46:43] ppocr INFO:         delimiter : |
[2023/09/18 11:46:43] ppocr INFO:         label_file_list : ['./data/50000/test_det_img.txt']       
[2023/09/18 11:46:43] ppocr INFO:         name : SimpleDataSet
[2023/09/18 11:46:43] ppocr INFO:         transforms :
[2023/09/18 11:46:43] ppocr INFO:             DecodeImage :
[2023/09/18 11:46:43] ppocr INFO:                 channel_first : False
[2023/09/18 11:46:43] ppocr INFO:                 img_mode : BGR
[2023/09/18 11:46:43] ppocr INFO:             DetLabelEncode : None
[2023/09/18 11:46:43] ppocr INFO:             DetResizeForTest :
[2023/09/18 11:46:43] ppocr INFO:                 image_shape : [736, 1280]
[2023/09/18 11:46:43] ppocr INFO:             NormalizeImage :
[2023/09/18 11:46:43] ppocr INFO:                 mean : [0.485, 0.456, 0.406]
[2023/09/18 11:46:43] ppocr INFO:                 order : hwc
[2023/09/18 11:46:43] ppocr INFO:                 scale : 1./255.
[2023/09/18 11:46:43] ppocr INFO:                 std : [0.229, 0.224, 0.225]
[2023/09/18 11:46:43] ppocr INFO:             ToCHWImage : None
[2023/09/18 11:46:43] ppocr INFO:             KeepKeys :
[2023/09/18 11:46:43] ppocr INFO:                 keep_keys : ['image', 'shape', 'polys', 'ignore_tags']
[2023/09/18 11:46:43] ppocr INFO:     loader :
[2023/09/18 11:46:43] ppocr INFO:         batch_size_per_card : 1
[2023/09/18 11:46:43] ppocr INFO:         drop_last : False
[2023/09/18 11:46:43] ppocr INFO:         num_workers : 4
[2023/09/18 11:46:43] ppocr INFO:         shuffle : False
[2023/09/18 11:46:43] ppocr INFO:         use_shared_memory : True
[2023/09/18 11:46:43] ppocr INFO: Global :
[2023/09/18 11:46:43] ppocr INFO:     cal_metric_during_train : False
[2023/09/18 11:46:43] ppocr INFO:     checkpoints : None
[2023/09/18 11:46:43] ppocr INFO:     distributed : False
[2023/09/18 11:46:43] ppocr INFO:     epoch_num : 200
[2023/09/18 11:46:43] ppocr INFO:     eval_batch_step : [0, 2000]
[2023/09/18 11:46:43] ppocr INFO:     infer_img : doc/imgs_en/img_10.jpg
[2023/09/18 11:46:43] ppocr INFO:     log_smooth_window : 20
[2023/09/18 11:46:43] ppocr INFO:     pretrained_model : ./models/MobileNetV3_large_x0_5_pretrained 
[2023/09/18 11:46:43] ppocr INFO:     print_batch_step : 10
[2023/09/18 11:46:43] ppocr INFO:     save_epoch_step : 200
[2023/09/18 11:46:43] ppocr INFO:     save_inference_dir : None
[2023/09/18 11:46:43] ppocr INFO:     save_model_dir : ./output/my_db_mv3/
[2023/09/18 11:46:43] ppocr INFO:     save_res_path : ./output/det_db/my_predicts_db.txt
[2023/09/18 11:46:43] ppocr INFO:     use_gpu : True
[2023/09/18 11:46:43] ppocr INFO:     use_mlu : False
[2023/09/18 11:46:43] ppocr INFO:     use_visualdl : False
[2023/09/18 11:46:43] ppocr INFO:     use_xpu : False
[2023/09/18 11:46:43] ppocr INFO: Loss :
[2023/09/18 11:46:43] ppocr INFO:     alpha : 5
[2023/09/18 11:46:43] ppocr INFO:     balance_loss : True
[2023/09/18 11:46:43] ppocr INFO:     beta : 10
[2023/09/18 11:46:43] ppocr INFO:     main_loss_type : DiceLoss
[2023/09/18 11:46:43] ppocr INFO:     name : DBLoss
[2023/09/18 11:46:43] ppocr INFO:     ohem_ratio : 3
[2023/09/18 11:46:43] ppocr INFO: Metric :
[2023/09/18 11:46:43] ppocr INFO:     main_indicator : hmean
[2023/09/18 11:46:43] ppocr INFO:     name : DetMetric
[2023/09/18 11:46:43] ppocr INFO: Optimizer :
[2023/09/18 11:46:43] ppocr INFO:     beta1 : 0.9
[2023/09/18 11:46:43] ppocr INFO:     beta2 : 0.999
[2023/09/18 11:46:43] ppocr INFO:     lr :
[2023/09/18 11:46:43] ppocr INFO:         learning_rate : 0.001
[2023/09/18 11:46:43] ppocr INFO:     name : Adam
[2023/09/18 11:46:43] ppocr INFO:     regularizer :
[2023/09/18 11:46:43] ppocr INFO:         factor : 0
[2023/09/18 11:46:43] ppocr INFO:         name : L2
[2023/09/18 11:46:43] ppocr INFO: PostProcess : 
[2023/09/18 11:46:43] ppocr INFO:     box_thresh : 0.6
[2023/09/18 11:46:43] ppocr INFO:     max_candidates : 1000
[2023/09/18 11:46:43] ppocr INFO:     name : DBPostProcess
[2023/09/18 11:46:43] ppocr INFO:     thresh : 0.3
[2023/09/18 11:46:43] ppocr INFO:     unclip_ratio : 1.5
[2023/09/18 11:46:43] ppocr INFO: Train :
[2023/09/18 11:46:43] ppocr INFO:     dataset :
[2023/09/18 11:46:43] ppocr INFO:         data_dir : ./data/50000/train
[2023/09/18 11:46:43] ppocr INFO:         delimiter : |
[2023/09/18 11:46:43] ppocr INFO:         label_file_list : ['./data/50000/train_det_img.txt']      
[2023/09/18 11:46:43] ppocr INFO:         name : SimpleDataSet
[2023/09/18 11:46:43] ppocr INFO:         ratio_list : [1.0]
[2023/09/18 11:46:43] ppocr INFO:         transforms :
[2023/09/18 11:46:43] ppocr INFO:             DecodeImage :
[2023/09/18 11:46:43] ppocr INFO:                 channel_first : False
[2023/09/18 11:46:43] ppocr INFO:                 img_mode : BGR
[2023/09/18 11:46:43] ppocr INFO:             DetLabelEncode : None
[2023/09/18 11:46:43] ppocr INFO:             IaaAugment :
[2023/09/18 11:46:43] ppocr INFO:                 augmenter_args :
[2023/09/18 11:46:43] ppocr INFO:                     args :
[2023/09/18 11:46:43] ppocr INFO:                         p : 0.5
[2023/09/18 11:46:43] ppocr INFO:                     type : Fliplr
[2023/09/18 11:46:43] ppocr INFO:                     args :
[2023/09/18 11:46:43] ppocr INFO:                         rotate : [-10, 10]
[2023/09/18 11:46:43] ppocr INFO:                     type : Affine
[2023/09/18 11:46:43] ppocr INFO:                     args :
[2023/09/18 11:46:43] ppocr INFO:                         size : [0.5, 3]
[2023/09/18 11:46:43] ppocr INFO:                     type : Resize
[2023/09/18 11:46:43] ppocr INFO:             EastRandomCropData :
[2023/09/18 11:46:43] ppocr INFO:                 keep_ratio : True
[2023/09/18 11:46:43] ppocr INFO:                 max_tries : 50
[2023/09/18 11:46:43] ppocr INFO:                 size : [640, 640]
[2023/09/18 11:46:43] ppocr INFO:             MakeBorderMap :
[2023/09/18 11:46:43] ppocr INFO:                 shrink_ratio : 0.4
[2023/09/18 11:46:43] ppocr INFO:                 thresh_max : 0.7
[2023/09/18 11:46:43] ppocr INFO:                 thresh_min : 0.3
[2023/09/18 11:46:43] ppocr INFO:             MakeShrinkMap :
[2023/09/18 11:46:43] ppocr INFO:                 min_text_size : 8
[2023/09/18 11:46:43] ppocr INFO:                 shrink_ratio : 0.4
[2023/09/18 11:46:43] ppocr INFO:             NormalizeImage :
[2023/09/18 11:46:43] ppocr INFO:                 mean : [0.485, 0.456, 0.406]
[2023/09/18 11:46:43] ppocr INFO:                 order : hwc
[2023/09/18 11:46:43] ppocr INFO:                 scale : 1./255.
[2023/09/18 11:46:43] ppocr INFO:                 std : [0.229, 0.224, 0.225]
[2023/09/18 11:46:43] ppocr INFO:             ToCHWImage : None
[2023/09/18 11:46:43] ppocr INFO:             KeepKeys :
[2023/09/18 11:46:43] ppocr INFO:                 keep_keys : ['image', 'threshold_map', 'threshold_mask', 'shrink_map', 'shrink_mask']
[2023/09/18 11:46:43] ppocr INFO:     loader :
[2023/09/18 11:46:43] ppocr INFO:         batch_size_per_card : 2
[2023/09/18 11:46:43] ppocr INFO:         drop_last : False
[2023/09/18 11:46:43] ppocr INFO:         num_workers : 4
[2023/09/18 11:46:43] ppocr INFO:         shuffle : True
[2023/09/18 11:46:43] ppocr INFO:         use_shared_memory : True
[2023/09/18 11:46:43] ppocr INFO: profiler_options : None
[2023/09/18 11:46:43] ppocr INFO: train with paddle 2.5.1 and device Place(gpu:0)
[2023/09/18 11:46:43] ppocr INFO: Initialize indexs of datasets:['./data/50000/train_det_img.txt']  
'MakeBorderMap'
[2023/09/18 11:46:43] ppocr INFO: Initialize indexs of datasets:['./data/50000/test_det_img.txt']
W0918 11:46:43.299890 14776 gpu_resources.cc:96] The GPU architecture in your current machine is Pascal, which is not compatible with Paddle installation with arch: 70 75 80 86 , it is recommended to 
install the corresponding wheel package according to the installation information on the official Paddle website.
W0918 11:46:43.299890 14776 gpu_resources.cc:119] Please NOTE: device: 0, GPU Compute Capability: 6.1, Driver API Version: 12.2, Runtime API Version: 12.0
W0918 11:46:43.299890 14776 gpu_resources.cc:149] device: 0, cuDNN Version: 8.9.
[2023/09/18 11:46:43] ppocr INFO: train dataloader has 20000 iters
[2023/09/18 11:46:43] ppocr INFO: valid dataloader has 10000 iters
[2023/09/18 11:46:43] ppocr INFO: load pretrain successful from ./models/MobileNetV3_large_x0_5_pretrained
[2023/09/18 11:46:43] ppocr INFO: During the training process, after the 0th iteration, an evaluation is run every 2000 iterations
(ppocr) F:\python_project\PaddleOCR>

**Comment by jiangbossB** [comment]
(ppocr) F:\python_project\PaddleOCR>pip list
Package               Version
--------------------- -------------
aiohttp               3.8.5
aiosignal             1.3.1
annotated-types       0.5.0
anyio                 3.7.1
astor                 0.8.1
async-timeout         4.0.3
attrdict              2.0.1
attrdict3             2.0.2
attrs                 23.1.0
Babel                 2.12.1
bce-python-sdk        0.8.87
beautifulsoup4        4.12.2
blinker               1.6.2
cachetools            5.3.1
certifi               2022.12.7
charset-normalizer    3.1.0
click                 8.1.6
colorama              0.4.6
colorlog              6.7.0
contourpy             1.1.0
cssselect             1.2.0
cssutils              2.7.1
cycler                0.11.0
Cython                3.0.0
datasets              2.14.5
decorator             5.1.1
dill                  0.3.4
et-xmlfile            1.1.0
exceptiongroup        1.1.2
fastapi               0.103.1
filelock              3.12.3
fire                  0.5.0
Flask                 2.3.2
flask-babel           3.1.0
fonttools             4.42.0
frozenlist            1.4.0
fsspec                2023.6.0
future                0.18.3
h11                   0.14.0
httpcore              0.17.3
httpx                 0.24.1
huggingface-hub       0.16.4
idna                  3.4
imageio               2.31.1
imgaug                0.4.0
importlib-metadata    6.8.0
importlib-resources   6.0.1
install               1.3.5
itsdangerous          2.1.2
jieba                 0.42.1
Jinja2                3.1.2
joblib                1.3.2
kiwisolver            1.4.4
lazy_loader           0.3
lmdb                  1.4.1
lxml                  4.9.3
markdown-it-py        3.0.0
MarkupSafe            2.1.3
matplotlib            3.7.2
mdurl                 0.1.2
multidict             6.0.4
multiprocess          0.70.12.2
networkx              3.1
numpy                 1.25.2
onnx                  1.14.1
opencv-contrib-python 4.6.0.66
opencv-python         4.6.0.66
openpyxl              3.1.2
opt-einsum            3.3.0
packaging             23.1
paddle-bfloat         0.1.7
paddle2onnx           1.0.6
paddlefsl             1.1.0
paddlenlp             2.4.5
paddleocr             2.7.0.2
paddlepaddle-gpu      2.5.1.post120
pandas                2.0.3
pdf2docx              0.5.6
Pillow                9.5.0
pip                   23.1.2
PPOCRLabel            2.1.3
premailer             3.10.0
protobuf              3.20.0
psutil                5.9.5
pyarrow               13.0.0
pyclipper             1.3.0.post4
pycryptodome          3.18.0
pydantic              2.3.0
pydantic_core         2.6.3
Pygments              2.16.1
pymssql               2.2.7
PyMuPDF               1.20.2
pypandoc              1.11
pyparsing             3.0.9
PyQt5                 5.15.9
PyQt5-Qt5             5.15.2
PyQt5-sip             12.12.2
python-dateutil       2.8.2
python-docx           0.8.11
pytz                  2023.3
PyWavelets            1.4.1
PyYAML                6.0
rapidfuzz             3.2.0
rarfile               4.0
requests              2.29.0
rich                  13.5.2
safetensors           0.3.3
scikit-image          0.21.0
scikit-learn          1.3.0
scipy                 1.11.1
sentencepiece         0.1.99
seqeval               1.2.2
setuptools            68.0.0
shapely               2.0.1
six                   1.16.0
sniffio               1.3.0
soupsieve             2.4.1
SQLAlchemy            1.4.0
starlette             0.27.0
termcolor             2.3.0
threadpoolctl         3.2.0
tifffile              2023.8.12
tqdm                  4.66.1
typer                 0.9.0
typing_extensions     4.7.1
tzdata                2023.3
urllib3               1.26.15
uvicorn               0.23.2
visualdl              2.5.3
Werkzeug              2.3.6
wheel                 0.38.4
xlrd                  1.2.0
xxhash                3.3.0
yacs                  0.1.8
yarl                  1.9.2
zipp                  3.16.2

**Comment by jiangbossB** [comment]
release 2.7
